app/src/offline/java/com/kamron/pogoiv/CrashlyticsWrapper.java

```diff
@@ -18,6 +19,8 @@ public CrashReportingTree(Context context) {
         @Override
         protected void log(int priority, String tag, String message, Throwable t) {
             // there is no logging of crash reports in offline builds
+            // XXX we use error level but should look at the priority.
+            Log.e(tag, message, t);
```

Comment:
This will log everything as an error regardless of the actual log level.

You probably want to do `Log.println(priority, tag, message);` and concatenate the stacktrace in yourself. 

Or maybe something like: 
```
        if (priority == Log.ASSERT) {
          Log.wtf(tag, message);
        } else {
          Log.println(priority, tag, message);
        }
```

There is a character limit for logs too though I don't know much about it